### PR TITLE
Import .pcap RADIUS tests from wireshark project

### PR DESCRIPTION
**Pull request by jpereira**
Copied from https://wiki.wireshark.org/SampleCaptures and converted
with:

	./scripts/pcap2decode-proto.py -p radius -f ~/radius_localhost.pcapng \
		> src/tests/unit/protocols/radius/packet_wireshark01.txt

### CODE REVIEW

File: src/tests/unit/protocols/radius/packet_wireshark01.txt [hunk 1]
```diff
@@ -0,0 +1,797 @@
...
+#    |  type      = NAS-Port
+#    |  len       = 6
+#    |  value     = 123
+#    |[ Message-Authenticator ]
+#    |  type      = Message-Authenticator
+#    |  len       = 18
+#    |  value     = aa5dcc5ab695fb6baf1cec851f601e22
+#
+decode-proto 01 2b 00 4b 6f 75 df 88 62 88 8f 90 d4 f6 ff 19 ae 4f 38 fb 01 07 73 74 65 76 65 02 12 1a 6c 45 1b c3 a8 45 7e 4a 50 39 e3 6c 48 43 b6 04 06 c0 a8 00 1c 05 06 00 00 00 7b 50 12 aa 5d cc 5a b6 95 fb 6b af 1c ec 85 1f 60 1e 22
+match Packet-Type = Access-Request, Packet-Authentication-Vector = 0x6f75df8862888f90d4f6ff19ae4f38fb, User-Name = "steve", User-Password = "\214\202LѷU?dZ\026>\307l\\\301=", NAS-IP-Address = 192.168.0.28, NAS-Port = 123, Message-Authenticator = 0xaa5dcc5ab695fb6baf1cec851f601e22
```
Comment (arr2036) [inline src/tests/unit/protocols/radius/packet_wireshark01.txt]: We need to know the secret

**Comment by arr2036** [comment]
We need the secret and the ability to set it for a test file

File: src/tests/unit/protocols/radius/packet_wireshark01.txt (hunk 1 above)
Comment (jpereira) [reply src/tests/unit/protocols/radius/packet_wireshark01.txt]: The secret is `testing123`

**Comment by jpereira** [comment]
The secret is `testing123`. But, probably it will be necessary to do some extra pokes on the unit_test to handle it. Or perhaps not?

It was extracted from https://wiki.wireshark.org/SampleCaptures

image

**Comment by alandekok** [comment]
We can add a separate way to set the secret.  Likely via a new attribute.  I spent some time looking at ways to pass things from the test harness to the test tools.  The options here are limited.

**Pull request merged**